tools/Datadog.Core.Tools/DependencyHelpers.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.IO;
+using Datadog.Core.Tools;
+
+namespace PrepareRelease.Tools
+{
+    public static class DependencyHelpers
+    {
+        private const string RequiredBuildConfig = "Release";
+
+        public static string[] GetTracerReleaseBinContent(string frameworkMoniker, string extension = "dll")
```

Comment:
nit: You could probably even do some more code-sharing between these two methods

Reply:
For sure